wp-content/themes/core/components/container/Controller.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\container;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+
+class Controller extends Abstract_Controller {
+	/**
+	 * @var string
+	 */
+	private $tag;
+	/**
+	 * @var string[]
+	 */
+	private $classes;
+	/**
+	 * @var string[]
+	 */
+	private $attrs;
+	/**
+	 * @var string|Deferred_Component
+	 */
+	private $content;
+
+	public function __construct( array $args = [] ) {
```

Comment:
This makes me very happy after doing this on every component controller. @ryanurban This is where we should align around common methods for managing classes and other consistent parts of a component.